getting-started/vs-integration/dpl-wizard.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Using the Document Processing Libraries Wizard
```

Comment:
```suggestion
title: Using the Document Processing Libraries
```
Remove the "Wizard" word from the titles for consistency. The other article titles do not include it. Applies for page title as well.